### PR TITLE
Feature/2823/seven bridges

### PR DESCRIPTION
**Pull request by coverbeck**
Doc for CGC.

Preview version here: https://beta.docs.dockstore.org/en/feature-2823-seven-bridges/end-user-topics/cgc-launch-with.html

### CODE REVIEW

File: docs/end-user-topics/cgc-launch-with.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Cancer Genomics Cloud Launch-With
+=================================
+
+Dockstore integrates with the Cancer Genomics Cloud (CGC) platform, allowing you to launch
```
Comment (agduncan94) [inline docs/end-user-topics/cgc-launch-with.rst]: Should link to the CGC site.

File: docs/end-user-topics/cgc-launch-with.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Cancer Genomics Cloud Launch-With
+=================================
+
+Dockstore integrates with the `Cancer Genomics Cloud <https://cgc.sbgenomics.com>` (CGC)
```
Comment (denis-yuen) [inline docs/end-user-topics/cgc-launch-with.rst]: Link looks weird in preview at https://beta.docs.dockstore.org/en/feature-2823-seven-bridges/end-user-topics/cgc-launch-with.html

File: docs/end-user-topics/cgc-launch-with.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+.. figure:: /assets/images/docs/cgc/cgc_from_dockstore.png
+   :alt: CWL workflow
+
+   CWL workflow
+
+If not logged into the CGC, you will be prompted to login. Otherwise, or
+after login, you will be presented with the following screen.
+
+.. figure:: /assets/images/docs/cgc/cgc_from_dockstore2.png
```
Comment (denis-yuen) [inline docs/end-user-topics/cgc-launch-with.rst]: App URL in the figure doesn't match the first picture at all (looks like h3agatk vs UM aligner)

File: docs/end-user-topics/cgc-launch-with.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+   :alt: CWL workflow import
+
+   CWL workflow import
+
+You will need to pick a project to export it into. The project
+must already exist.
+
+Then hit the "Import" button and continue from within the Terra
```
Comment (denis-yuen) [inline docs/end-user-topics/cgc-launch-with.rst]: Terra interface

File: docs/end-user-topics/cgc-launch-with.rst (hunk 2 above)
Comment (coverbeck) [reply docs/end-user-topics/cgc-launch-with.rst]: Fixed

File: docs/end-user-topics/cgc-launch-with.rst (hunk 4 above)
Comment (coverbeck) [reply docs/end-user-topics/cgc-launch-with.rst]: Fixed

File: docs/end-user-topics/cgc-launch-with.rst (hunk 3 above)
Comment (coverbeck) [reply docs/end-user-topics/cgc-launch-with.rst]: Fixed

File: docs/end-user-topics/cgc-launch-with.rst (hunk 1 above)
Comment (coverbeck) [reply docs/end-user-topics/cgc-launch-with.rst]: Fixed

**Pull request merged**